concordium-sdk/src/main/java/com/concordium/sdk/transactions/smartcontracts/WasmModuleSource.java

```diff
@@ -0,0 +1,71 @@
+package com.concordium.sdk.transactions.smartcontracts;
+
+import com.concordium.sdk.types.UInt32;
+import lombok.val;
+import org.apache.commons.codec.DecoderException;
+import org.apache.commons.codec.binary.Hex;
+
+import java.nio.ByteBuffer;
+import java.util.Objects;
+
+public abstract class WasmModuleSource {
+
+    /**
+     * The raw bytes of the WasmModule.
+     */
+    private final byte[] rawSource;
+
+    protected WasmModuleSource(byte[] rawModuleSource) {
+        this.rawSource = rawModuleSource;
+    }
+
+    public static WasmModuleSource createNew(WasmModuleVersion version, String hexRawModuleSource) {
```

Comment:
I'm not sure why it should be created via a hex encoded source? Why not let it simply be the bytes? 
I know that I did that in the example long time ago, but the purpose of the example was to show where in the code you needed to look for adding a new kind of transaction.

Reply:
done